frontend/src/rootReducer.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+import { combineReducers } from 'redux';
+
+import navigation from 'core/navigation';
+import entitieslist from 'modules/entitieslist';
+
+
+// Combine reducers from all modules, using their
+// NAME constant as key.
+const rootReducer = combineReducers({
+    [entitieslist.constants.NAME]: entitieslist.reducer,
+    [navigation.constants.NAME]: navigation.reducer,
+});
+
+export default rootReducer;
```

Comment:
Nit: you don't have to bind the combined reducere to `const rootReducer` if you only `export default` it right after. `export default combineReducers(...)` will work too. (This applies to `store.js` as well.)